website/sidebars.js

```diff
@@ -112,6 +112,11 @@ module.exports = {
                                     label: 'Gradual Rollout',
                                     id: 'feature-flag-tutorials/use-cases/gradual-rollout',
                                 },
+                                {
+                                    type: 'doc',
+                                    label: 'Feature flags for AI',
```

Comment:
I'll probably update the nav in one go at some point. Until then, for consistency:

```suggestion
                                    label: 'Feature Flags for AI',
```